docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<p>Please replace this sentence with:</p><p> </p><p>"As shown in this figure, the motifs discovered by the matrix profile (of <code>T_p</code>) does not reveal our two desirble subsequences, the ones that were shown in the previous figure. "</p><p> </p><p> </p><p>I think this can help readers understand why we need to use contrast profile technique later to discover the two desirbale subsequences.</p><p> </p><p>Please revise other figures accordingly if needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>